sphinx/ext/inheritance_diagram.py

```diff
@@ -412,13 +412,16 @@ def html_visit_inheritance_diagram(self: HTML5Translator, node: inheritance_diag
     pending_xrefs = cast(Iterable[addnodes.pending_xref], node)
     for child in pending_xrefs:
         if child.get('refuri') is not None:
-            if graphviz_output_format == 'SVG':
-                urls[child['reftitle']] = "../" + child.get('refuri')
+            # Construct the name from the URI if the reference is external via intersphinx
+            if child.get('internal', True):
```

Comment:
Is it safe to assume internal if not specified? (More directly, which will lead to fewer broken links?)

Reply:
I don't know if it's even possible for this code to see a child that doesn't have `internal` defined, but I coded it this way to be safer about preserving the existing behavior just in case there are edge cases.  That is, I only extract the name from `refuri` if both `internal` is defined and is `False`.  It might be clearer for future maintenance if I flipped the two branches:
```python
            # Construct the name from the URI if the reference is external via intersphinx
            if not child.get('internal', True):
                refname = child['refuri'].rsplit('#', 1)[-1]
            else:
                refname = child['reftitle']
```
Thoughts?

Reply:
Yes, I think this would be slightly clearer -- it shows that the default is to not use ``refuri``.

Reply:
Okay, flipped for clarity